src/App.js

```diff
@@ -41,10 +37,15 @@ function isLoggedIn() {
 
 export default App;
 
-const AuthenticateUser = ({ children, redirectPath = '/login' }) => {
+const AuthenticateUser = ({ children, redirectPath = '/login', setSearchInput }) => {
```

Comment:
In  a future PR, this could perhpas be moved in to it's own dedicated component now.